### ISSUE TITLE
Error building `examples/bzlmod`

### ISSUE THREAD

**Issue by yujunz** [seh/rules_cue#8]
```
cd examples/bzlmod

bazel build //root
ERROR: Skipping '//root': error loading package 'root': Unable to find package for @bazel_skylib//rules:write_file.bzl: The repository '@bazel_skylib' could not be resolved: Repository '@bazel_skylib' is not defined.
WARNING: Target pattern parsing failed.
ERROR: error loading package 'root': Unable to find package for @bazel_skylib//rules:write_file.bzl: The repository '@bazel_skylib' could not be resolved: Repository '@bazel_skylib' is not defined.
INFO: Elapsed time: 0.061s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
    currently loading: root
```

**Comment by seh** [comment]
How strange: I only just saw this issue today. I'm not sure why I'm not subscribed automatically to issues against my own repository. I'm looking into that now.

I have a patch ready for merging that sets this repository to using the recently released Bazel version 7.0.0, which enables use of Bazel modules by default. As part of it, I removed the top-level _.bazelrc_ file, having found it now empty after finally removing the command-line flag to enable use of Bazel modules. Given that, I don't think that adding a second _.bazelrc_ file as you proposed in #9 will be necessary.
